core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -517,6 +517,9 @@ private[spark] class MesosClusterScheduler(
           .addAllResources(memResourcesToUse.asJava)
         offer.resources = finalResources.asJava
         submission.schedulerProperties.get("spark.mesos.executor.docker.image").foreach { image =>
+          val forcePullImage = submission.schedulerProperties
```

Comment:
I don't see any changes to CoarseMesosSchedulerBackend.  It's not listed in one of the modified files.